app/controllers/api/projects.php

```diff
@@ -374,9 +376,10 @@
     ->param('legalCity', '', new Text(256), 'Project legal city. Max length: 256 chars.', true)
     ->param('legalAddress', '', new Text(256), 'Project legal address. Max length: 256 chars.', true)
     ->param('legalTaxId', '', new Text(256), 'Project legal tax ID. Max length: 256 chars.', true)
+    ->param('authDuration', 525600, new Range(0, 525600), 'Project session length in minutes. Max length: 525600 minutes.', true)
     ->inject('response')
     ->inject('dbForConsole')
-    ->action(function (string $projectId, string $name, string $description, string $logo, string $url, string $legalName, string $legalCountry, string $legalState, string $legalCity, string $legalAddress, string $legalTaxId, Response $response, Database $dbForConsole) {
+    ->action(function (string $projectId, string $name, string $description, string $logo, string $url, string $legalName, string $legalCountry, string $legalState, string $legalCity, string $legalAddress, string $legalTaxId, int $authDuration, Response $response, Database $dbForConsole) {
```

Comment:
@PineappleIOnic this param should be updated by a standalone route just like we have for other auth settings. We should remove it from the project creation endpoint.

Reply:
It should replicate this method, same for how we store the setting, no need for a dedicated column.
https://github.com/appwrite/appwrite/blob/842185e45d84f5fcfcc2330f3dfa680fe61504e9/app/controllers/api/projects.php#L471-L500